### ISSUE TITLE
Display of full text content and/or URLs

### ISSUE THREAD

**Issue by mdlincoln** [cmu-lib/dhweb_app#351]
Right now, any full text content we have entered into the DB (even if we know it's one of the ADHO conferences with CC license) is hidden from public users. Some questions:

Are there any cases, such as when we have marked the text as CC licensed, that we would display the full text? 

Even if public users cannot see the full text in our database, they can still _search_ the full text. There's a "full-text available" badge on the abstract search results that right now only displays to logged-in users. Should we display this to public users, if only to let them know that the search also goes through indexed full text? They'd at least be able to click through via the URL attached to the abstract.

All those abstract URLs are public, yes?

**Comment by scottbot** [comment]
Yes, those abstract URLs are public - that search feature is a good idea I had not considered. In that case, we would need a note saying something like "the text for this abstract exists, but we cannot display it here because it is protected by copyright. If you would like access to the full text, please click the below link (if it exists), or find the conference proceedings through your institution's library."

Maybe instead of trying to make a taxonomy of licenses (esp. if there is no license, but we get explicit permission to share from the original rights owner), we have an additional toggle at the conference level, that Nickoal or I can switch, that says whether the abstracts can be shared publicly?

**Comment by mdlincoln** [comment]
- [ ] add `full_text_public` boolean to conferences (including to conference edit/create form)
- [ ] change `work_detail` template

**Comment by mdlincoln** [comment]
For those abstracts where there is a specific license, the title gets displayed (like the bottom of the page at https://dh-abstracts.library.cmu.edu/works/6414)

What do you want to display for conferences where you don't have a specific license, but you're publishing the text anyway?

**Comment by scottbot** [comment]
"This text is republished here with permission from the original rights holder."

**Comment by mdlincoln** [comment]
Alright. What I may do is add that as a license in the database (that way if you ever need to change the precise wording, you can without needing to muck in the code) and then make it the default license for any abstract whose conference is set to be public.

**Comment by mdlincoln** [comment]
Alright, I've deployed this update. When you edit a conference now, you'll have a checkbox to mark the conference as "full text public". This will assign all its abstracts the new default license (unless that abstract already has a license like CC-BY which is assigned to a bunch of the ADHO ones we imported way back.)

Likewise, when you add a new work, if its conference is set to "full text public" then it will get the default license assigned automatically, no change needed in the way that students enter the data.
